python/pyhie/allspark/model/qwen_v20_moe.py

```
@@ -594,34 +640,78 @@ def _load_and_permute(name):
+                                scale_list = [None] * len(torch_name)
+                                for f in sfuts:
+                                    scale_list[sfuts[f]] = f.result()
+                            if all(s is not None for s in scale_list):
+                                scale_inv = torch.stack(scale_list).cpu()
+                            del scale_list
                     else:
                         expert_tensors = []
+                        scale_list = []
                         for name in torch_name:
                             expert_tensors.append(
                                 torch.permute(torch_weight[name], (1, 0)).contiguous())
```

Flake8 (7.3.0)

[error] 'torch' may be undefined, or defined from star imports: ..quantization, .model_base, .quantization_utils

(F405)